### ISSUE TITLE
Pokedrops not form exclusive

### ISSUE THREAD

**Issue by Deathnight7976** [NebulaMC-GG/Support#1638]
Burmy drops items from all 3 of its forms resulting in green burmys dropping things like sand and iron.

Screenies:
![burmy](https://user-images.githubusercontent.com/17133410/178589539-248875c6-b790-4dd4-93c0-eb3f4d883f04.png)

![burmydrops](https://user-images.githubusercontent.com/17133410/178589578-bb3066c7-be78-4f35-a651-b23afed2e36e.png)

This is also true for other pokemon such as:
Lycanroc
Ponyta

Basically any pokemon that has multiple forms.

**Comment by Rasgnarok** [comment]
This is now fixed as of 9.0.6.
